### ISSUE TITLE
Feature Request (Backlog): Emojis/ Smileys in Survey force for Rating

### ISSUE THREAD

**Issue by jrattanpal** [SalesforceLabs/survey-force#124]
https://trailblazer.salesforce.com/ideaView?id=0874V000000UKxDQAW

When sending a feedback survey form, I need the rating to be as smileys/emojis. Since Survey force does not support the feature of smileys, what is other alternative to resolve this issue within Surveyforce?Will this feature be available in future?
--

**Comment by dschach** [comment]
If rating is limited to a specific number of options (maybe 5) then I suppose we could let the user put in a URL for each of the five ratings. But I don't know how we'd build the survey with this, so it may be too much for us to do.
At this point, buy Salesforce Surveys :)

**Comment by jrattanpal** [comment]
Agreed. I added this as it came from AppExchange. But I set the expectation that this may never happen. 

I also wanted to see if, somehow, an idea will magically appear to easily solve this.
